public/app/core/components/PasswordField.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, useState } from 'react';
+import { selectors } from '@grafana/e2e-selectors';
+
+import { Input, IconButton } from '@grafana/ui';
+
+export interface Props {
+  autoFocus?: boolean;
+  autoComplete?: string;
+  id?: string;
+  passwordHint?: string;
+  register: any;
+}
+
+export const PasswordField: FC<Props> = ({ autoComplete, autoFocus, id, passwordHint, register }) => {
```

Comment:
```suggestion
export const PasswordField: FC<Props> = ({ autoComplete, autoFocus, id, passwordHint, ...props }) => {
```
Instead of defining `register` prop on the `PasswordField`, you can just pass the rest of them, so you can use the same signature of `{...register()}` on this component.

Reply:
addressed